docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Regression/OrdinaryLeastSquaresAdvanced.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using Microsoft.ML;
+using Microsoft.ML.Data;
+using Microsoft.ML.SamplesUtils;
+
+namespace Samples.Dynamic.Trainers.Regression
+{
+    public static class OrdinaryLeastSquaresAdvanced
+    {
+        // This example requires installation of additional nuget package <a href="https://www.nuget.org/packages/Microsoft.ML.Mkl.Components/">Microsoft.ML.Mkl.Components</a>.
+        // In this examples we will use the housing price dataset. The goal is to predict median home value.
+        // For more details about this dataset, please see https://archive.ics.uci.edu/ml/machine-learning-databases/housing/
+        public static void Example()
+        {
+            // Downloading a regression dataset from github.com/dotnet/machinelearning
+            string dataFile = Microsoft.ML.SamplesUtils.DatasetUtils.DownloadHousingRegressionDataset();
+
+            // Create a new ML context, for ML.NET operations. It can be used for exception tracking and logging, 
+            // as well as the source of randomness.
+            var mlContext = new MLContext(seed: 3);
+
+            // Creating a data loader, based on the format of the data
+            // The data is tab separated with all numeric columns.
+            // The first column being the label and rest are numeric features
+            // Here only seven numeric columns are used as features
+            var dataView = mlContext.Data.LoadFromTextFile(dataFile, new TextLoader.Options
+            {
+                Separators = new[] { '\t' },
+                HasHeader = true,
+                Columns = new[]
+               {
+                    new TextLoader.Column("Label", DataKind.Single, 0),
+                    new TextLoader.Column("Features", DataKind.Single, 1, 6)
+                }
+            });
```

Comment:
together with this

Reply:
I am not following. Could you elaborate?

---
In reply to: [275457799](https://github.com/dotnet/machinelearning/pull/3319#discussion_r275457799) [](ancestors = 275457799)